src/protocols/LoRaWAN/LoRaWAN.cpp

```diff
@@ -2504,7 +2351,7 @@ bool LoRaWANNode::execMacCommand(LoRaWANMacCommand_t* cmd, bool saveToEeprom) {
       float freq = (float)freqRaw/10000.0;
       uint8_t maxDr = (cmd->payload[4] & 0xF0) >> 4;
       uint8_t minDr = cmd->payload[4] & 0x0F;
-      RADIOLIB_DEBUG_PROTOCOL_PRINTLN("New channel: index = %d, freq = %f MHz, maxDr = %d, minDr = %d", chIndex, freq, maxDr, minDr);
+      // RADIOLIB_DEBUG_PROTOCOL_PRINTLN("NewChannelReq: index = %d, freq = %f MHz, maxDr = %d, minDr = %d", chIndex, freq, maxDr, minDr);
```

Comment:
Not sure why is this commented out?

Reply:
Mostly because a few lines down, the resulting new channel is printed, and the output then is almost exactly duplicated.. so it's just saving some clutter. But I did not completely remove that line.. any preference?

Reply:
If the information is captured somewhere, I would prefer to remove this line. Otherwise it creates confusion why is that commented-out line there. If it is needed, why is it commented out? And if it is not needed, why does it stay there? It's this weird middle-ground that creates room for interpretation.